src/player.cpp

```diff
@@ -10102,6 +10102,20 @@ int player::get_wind_resistance(body_part bp) const
     return totalCoverage;
 }
 
+int player::get_weight() const
+{
+    int ret = 0;
+    int wornWeight = 0;
+    for (auto itm : worn) {
+        wornWeight += itm.weight();
+    }
+    ret += Creature::get_weight(); // The base weight of the player's body
```

Comment:
Now that I think about it, `Creature::get_weight` should be a purely virtual method. Doesn't make sense to have a default case for an abstract class when every child class will overload it.